api_compatibility_policy.md

```diff
@@ -31,15 +31,19 @@ The `apiVersion` field in a Tekton CRD determines whether the overall API (and i
 
 Within a stable CRD, certain opt-in features or API fields gated may be considered `alpha` or `beta`. Similarly, within a beta CRD, certain opt-in features may be considered `alpha`. See the section on Feature Gates for details.
 
-The following CRDs are considered beta, though features may be introduced that are
-alpha:
+The following CRDs are considered stable, though features may be introduced that are
+alpha or beta:
 
 - `Task`
 - `TaskRun`
-- `ClusterTask`
 - `Pipeline`
 - `PipelineRun`
 
+`CustomRun` is considered a beta CRD. Adding new fields to `CustomRun` is considered a [backwards incompatible change](#backwards-incompatible-changes),
```

Comment:
Adding new fields to customrun forces custom run controllers to update to support them-- added in https://github.com/tektoncd/pipeline/pull/5621

Reply:
Highlighted the difference between the policy in https://github.com/tektoncd/pipeline/pull/5621 and this one -- what if we add new fields that are not required to be supported in the CustomRun controllers?

"Adding a new field to the CustomRun API **that all CustomRun controllers are expected to support**
is also a backwards incompatible change, as CustomRun controllers that were valid before the change
would be invalid after the change."

Reply:
ah I see what you're saying now-- I think in general we should probably not do that unless we have a very good reason (since it can just go in the custom spec). I was thinking of updating the design principles to propose this-- wdyt?

Re: this PR, is there a suggested wording that would make this more clear? Maybe just including the block you've highlighted?

Reply:
done, thanks for the feedback!